lib/transports/odbcTransport.js

```diff
@@ -28,6 +28,11 @@ try {
 }
 
 function odbcCall(config, xmlInput, done) {
+  // odbc transport is not available bail out
+  if (odbc === null) {
```

Comment:
For giggles I made a test program to confirm this remained null:

```js
let foo = null;
try {
    foo = require('bar');
} catch (e)
{
}
console.log(foo);
```

prints `null`, so seems to be ok!